doc/ref_cert/README.md

```diff
@@ -1,5 +1,43 @@
 [<< Back](../)
 
+# CNTT Reference Conformance (RC) Requirements and Testing Principles
+
+The objective each RC workstream is to provide an automated mechanism to measure either a network function (NF) or a cloud infrastructure with a standard set of requirements defined by the associated Reference Architecture (RA).  Through this measurement, a provider of either NFs or cloud infrastructure will be able to test their conformance to the RA.  This will ease the integration of NFs into operator environments that host compatible cloud infrastructures, thereby reducing cost, complexity, and time of integration.
```

Comment:
The first sentence is awkward -- what does it mean to "measure" a network function?

Reply:
how about "test"?

Reply:
```suggestion
The objective each RC workstream is to provide an automated mechanism to validate either a network function (NF) or a cloud infrastructure against a standard set of requirements defined by the associated Reference Architecture (RA).  Through this validation mechanism, a provider of either NFs or cloud infrastructure will be able to test their conformance to the RA.  This will ease the integration of NFs into operator environments that host compatible cloud infrastructures, thereby reducing cost, complexity, and time of integration.
```